packages/expect/src/__tests__/toThrowMatchers.test.ts

```diff
@@ -22,527 +22,522 @@ class CustomError extends Error {
   }
 }
 
-// `as const` needs newer babel which explodes on node 6
-const matchers: ['toThrowError', 'toThrow'] = ['toThrowError', 'toThrow'];
-
-matchers.forEach(toThrow => {
-  describe(toThrow, () => {
-    class Err extends CustomError {}
-    class Err2 extends CustomError {}
+describe.each(['toThrowError', 'toThrow'] as const)('%s', toThrow => {
```

Comment:
Oh.. Missed this one. The only change is these few lines above. The rest is just formatting of white space.